packages/hub/src/hub.ts

```diff
@@ -228,107 +227,83 @@ export class Hub implements HubInterface {
    * @inheritDoc
    */
   public addBreadcrumb(breadcrumb: Breadcrumb, hint?: BreadcrumbHint): void {
-    const top = this.getStackTop();
+    const { scope, client } = this.getStackTop();
 
-    if (!top.scope || !top.client) {
-      return;
-    }
+    if (!scope || !client) return;
 
     // eslint-disable-next-line @typescript-eslint/unbound-method
     const { beforeBreadcrumb = null, maxBreadcrumbs = DEFAULT_BREADCRUMBS } =
-      (top.client.getOptions && top.client.getOptions()) || {};
+      (client.getOptions && client.getOptions()) || {};
 
-    if (maxBreadcrumbs <= 0) {
-      return;
-    }
+    if (maxBreadcrumbs <= 0) return;
 
     const timestamp = dateTimestampInSeconds();
     const mergedBreadcrumb = { timestamp, ...breadcrumb };
     const finalBreadcrumb = beforeBreadcrumb
       ? (consoleSandbox(() => beforeBreadcrumb(mergedBreadcrumb, hint)) as Breadcrumb | null)
       : mergedBreadcrumb;
 
-    if (finalBreadcrumb === null) {
-      return;
-    }
+    if (finalBreadcrumb === null) return;
```

Comment:
Not a big fan of `if` without `{}`, a bit harder to read when scanning quickly through code, easier to introduce bugs. (isn't there a linter rule for this?!)

On top of that this change is introducing inconsistency in style. For example, later in this file:
![image](https://user-images.githubusercontent.com/88819/95325893-18479280-08a2-11eb-96cf-3f28ab7cbed2.png)